lms/resources/_js_config/__init__.py

```diff
@@ -205,6 +205,10 @@ def enable_lti_launch_mode(self, course, assignment: Assignment):
         # postMessage.
         self._config["hypothesisClient"] = self._hypothesis_client
 
+        self._config["hypothesisClient"][
+            "annotationMetadata"
```

Comment:
This is going to be called `metadata` for now. 

This is private/undocumented API detail at a the moment so we can change it later without much hassle.

Reply:
The actual name on this object, :shrug: this seemed like the most obvious.